app/src/functions/backup/index.ts

```diff
@@ -0,0 +1,13 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export const backupDynamoDB = {
+    handler: `${handlerPath(__dirname)}/handler.backupDynamoDB`,
+    description: "A function to create automated backups of the database tables.",
+    memorySize: 128,
+    timeOut: 5,
+    events: [
+        {
+            schedule: "rate(5 minutes)"
```

Comment:
looks good, I thought you were gonna use cron job instead of rate

Reply:
I'm going to switch to cron once I know that the backups are definitely going to be created. This is for testing.